.env.development

```diff
@@ -7,6 +7,6 @@ REACT_APP_LIBRARY_BACKEND=https://us-central1-excalidraw-room-persistence.cloudf
 REACT_APP_PORTAL_URL=http://localhost:3002
 # Fill to set socket server URL used for collaboration.
 # Meant for forks only: excalidraw.com uses custom REACT_APP_PORTAL_URL flow
-REACT_APP_WS_SERVER_URL=
+REACT_APP_WS_SERVER_URL=http://localhost:3002
```

Comment:
this actually we shouldn't commit :) — put into `.env.local`

Reply:
we aren't storing any user secrets in this file so should be fine to commit? That way Devs will not need to update it as well.

Reply:
ok, sounds reasonable. Changed the doc to reflect this